packages/fxa-shared/metrics/amplitude-event.1.schema.json

```diff
@@ -102,6 +102,36 @@
           "description": "Product ID of a subscription.",
           "type": "string",
           "maxLength": 128
+        },
+        "promotionCode": {
```

Comment:
Nit: This should be snake case to be consistent with the other field names and with the Tech Spec.

Reply:
done